scripts/release.sh

```diff
@@ -33,7 +37,7 @@ if [ -n "$(git status --porcelain)" ]; then
 fi
 
 echo Building, Testing, and Uploading Archives...
-./gradlew --no-parallel clean test install uploadArchives
+./gradlew --no-parallel clean test publishToMaven -Pstaging
```

Comment:
Currently, I do not see why you need the staging profile here. In the project there is no staging profile defined, is it?
Only in the gradle-example projects exist a staging profile to determine from where the plugin should be loaded.

Reply:
yes. Doing that rebase that late in the day probably wasn't a good idea